modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequestForm.testcase

```diff
@@ -106,7 +111,7 @@ definition {
 	@description = "LPS-162902 - Verify that can not submit the request form without fill all required fields"
 	@priority = 3
 	test CannotSubmitFormWithoutRequiredFields {
-		task ("When the user clicks on submit button in the request form without filling out any required fields") {
+		task ("When the Employee user clicks the send button on the request form without filling in a required field") {
```

Comment:
Please change this task to : `"When the Employee user clicks on submit button in the request form without filling in any required field"`